bin/test.sh

```diff
@@ -61,15 +64,24 @@ BOOKINFO_DEPLOYMENTS="details-v1 productpage-v1 ratings-v1 reviews-v1 reviews-v2
 if [ "$SKIP_SETUP" -ne 1 ]; then
     kubectl create ns bookinfo || true
     # We use the first namespace with sidecar injection enabled to determine the control plane's namespace.
-    ISTIO_CONTROL=$(kubectl get namespaces -o=jsonpath='{$.items[:1].metadata.labels.istio-env}' -l istio-env)
+    # Fails in KIND.
```

Comment:
which fails in kind?  Setup?  Creation of the namespace?  A TODO with issue might help offload some of the work to others.

Reply:
The command to get ISTIO_CONTROL - the test.sh was written for concourse ( another CI we want to support ).